src/libraries/Microsoft.PowerFx.Core/Parser/ParseFormulasResult.cs

```diff
@@ -78,16 +83,23 @@ internal class UDF
 
         internal bool IsImperative { get; }
 
-        internal bool NumberIsFloat { get; }
+        internal bool NumberIsFloat { get; }
+
+        /// <summary>
+        /// False if UDF is incomplete eg: Add(a: Number, b: Number): .
+        /// </summary>
+        internal bool IsValid { get; }
```

Comment:
>IsValid [](http://example.com/codeflow?start=22&length=7)

nit - can we make this name specific. 
Is it just about parsing? Lots of other place sto get errors...

`IsParseValid`?

Reply:
This is mainly about parsing